### DISCUSSION TITLE
How to start the psst-gui binary without open the terminal?

### DISCUSSION THREAD

**Discussion by vuon9** [jpochyla/psst#239] [Q&A] [answered]
As the title, I would like to open the app independently so it should n't open the terminal with log. I used the cargo build with release but it happens. Could I?

**Comment by jpochyla** [comment, marked as answer]
Hi, we're building the app bundles in the GitHub Actions script. In case you want do build them yourself, please follow the steps here (for the appropriate platform): https://github.com/jpochyla/psst/blob/master/.github/workflows/build.yml

**Reply by vuon9** [reply]
ah, I see. Thank you!
